dtfs-central-api/src/repositories/utilisation-reports-repo/utilisation-report.repo.ts

```diff
@@ -236,31 +236,4 @@ export const UtilisationReportRepo = SqlDbDataSource.getRepository(UtilisationRe
       },
     });
   },
-
-  /**
-   * Finds a utilisation report with the supplied id and attaches
-   * all the fee records which match the supplied facility id
-   * query with the payments attached and all the fee records attached
-   * to the payments
-   * @param reportId - The report id
-   * @param feeRecordFacilityIdQuery - The fee record statuses to filter by
-   * @returns The utilisation report with attached fee records and payments
-   */
-  async findOneByIdWithFeeRecordsFilteredByPartialFacilityId(reportId: number, feeRecordFacilityIdQuery: string): Promise<UtilisationReportEntity | null> {
```

Comment:
This method has been removed because we need to get every fee record to display both the premium payments table and the keying sheets table even if we have a facility id to filter by